extensions/amp-youtube/1.0/component.js

```diff
@@ -140,10 +170,10 @@ export function Youtube({
         '*'
       );
     }
-    if (data.event == 'infoDelivery' && playerState != undefined) {
+    if (event == 'infoDelivery' && playerState != undefined) {
```

Comment:
Can we pull out some of these string literals into an event enum?

Reply:
I don't think that would be helpful. These are localized in this handler and shouldn't be pulled out, ideally they should be checked once. An enum's name is disconnected from the actual event's name and that makes debugging harder.

Reply:
Sounds reasonable to me, thanks!